### PR TITLE
Update template, 201-web-app-sql-database, to comply schema definition:

### PR DESCRIPTION
**Pull request by justinyoo**
Hi, Team.

I've updated the template, 201-web-app-sql-database, to comply JSON schema definition as of 2015-01-01, with following details:

- Updated API versions for each resource
- Updated casing of resource types

I confirm that the changes I made were validated through Azure PowerShell cmdlet, `Test-AzureRmResourceGroupDeployment`. Please have a review and accept it.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@justinyoo__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: 201-web-app-sql-database/metadata.json [hunk 1]
```diff
@@ -2,6 +2,6 @@
   "itemDisplayName": "Provision a Web App with a SQL Database",
   "description": "This template provisions a Web App, a SQL Database, AutoScale settings, Alert rules, and App Insights. It configures a connection string in the web app for the database.",
   "summary": "Provision a Web App with a SQL Database",
-  "githubUsername": "tfitzmac",
-  "dateUpdated": "2015-08-11"
+  "githubUsername": "justinyoo",
```
Comment (singhkay) [inline 201-web-app-sql-database/metadata.json]: Please leave the original author intact here, date change is good

**Comment by singhkay** [comment]
@tfitzmac Can you review this PR?

**Comment by justinyoo** [comment]
@singhkay According to the `metadata.json` section on [readme.md](../blob/master/README.md), it's a little bit unclear whether the githubUsername should go to the original author or can be changed to the last modifier. Can we change that part to be more specific like:

### githubUsername ###

* Username must be the same as the username of the author submitting the Pull Request.
* Event if  any modification is made by others, this should remain as the original author.
* This is used to display template author and Github profile pic in the Azure.com index

I know it might be a big policy change, but it's worth having a discussion at some point, not necessarily right now.

**Comment by tfitzmac** [comment]
@justinyoo thank you for making these changes. When I tried deploying the template, I received an error for the firewall rules stating bad request. I looked at the schema and it looks like the property Version was removed for firewall rules. When I tried your template again but without Version, it deployed correctly. Can you remove Version from the properties section of firewallrules?

**Comment by justinyoo** [comment]
@tfitzmac Thanks, Tom. I've removed the version property from the firewallrules type. I also confirm that it's deployed without an issue.

**Comment by justinyoo** [comment]
@singhkay @tfitzmac Could you let me know when this will be reviewed again?

**Comment by tfitzmac** [comment]
I think it is ready to be merged.

**Comment by singhkay** [comment]
@tfitzmac Thanks for the update, merged

@justinyoo Re: Username, definitely. We're thinking about adding a comma separated list to the githubUsername or just pick up various contributors to the template from git history for Azure.com index.

**Pull request merged**